src/services/has-many-getter.js

```diff
@@ -7,12 +7,13 @@ const FiltersParser = require('./filters-parser');
 const OBJECTID_REGEXP = /^[0-9a-fA-F]{24}$/;
 
 class HasManyGetter {
-  constructor(parentModel, childModel, opts, params) {
+  constructor(parentModel, model, opts, params, user) {
     this._parentModel = parentModel;
-    this._childModel = childModel;
+    this._model = model;
```

Comment:
[MISC] Why have you changed this variable name?

Reply:
to match forest-express-sequelize